util__deploy_to_maven_central_remove_staged_artifact.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Could you use _strict mode_ and [shellcheck](https://github.com/koalaman/shellcheck/releases/tag/v0.4.7) for all scripts?

Reply:
Yes! I should have installed that earlier really... 
Shellcheck is now happy.